### PR TITLE
Adding the Work and About me sections

### PR DESCRIPTION
**Pull request by selma-belhadj**
In this PR, I have done the following tasks:

- Building the middle sections of the mobile website which include:
1. The "Works" section
2. The "About myself" section
- Using CSS Grid and Flexbox combined
- Setting up the Html and CSS linters

### CODE REVIEW

**Review by t-yanick — CHANGES_REQUESTED** [review]
Hi @selma-belhadj , 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a project, that does not passes all linters.

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏


Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html [hunk 1]
```diff
@@ -31,9 +31,133 @@ <h2 class="intro">Hey there. I'm Selma <br><span>I am a software<br> developer</
...
+            <li>Bootstrap</li>
+            <li>Ruby on rails</li>
+            <li>RSpec</li>
+            <li>Capybara</li> 
+            <li>Selenium</li> 
+        </ul>
+    </div>
+    <div class="about-div">
+        <img class="about-icon" src="images/skills.svg" alt="">
+        <h4>Skills</h4>
```
Comment (t-yanick) [inline index.html:151]: - [ ] In the same light, visit your CSS and give enough margin between the skills icon and the h4 text. See screenshots for clarity. One of the objectives of this project is to be able to reproduce a given design exactly. We shall achieve exactly that.

Your design 
image

Figma design 
image

File: styles.css [hunk 2]
```diff
@@ -90,3 +105,184 @@ h2 span {
...
+  color: #28352f;
+}
+
+#main-head h2 {
+  font-size: 28px;
+}
+
+.work-div {
+  text-align: center;
+  display: flex;
```
Comment (t-yanick) [inline styles.css:124]: [Optional] For best CSS practices and to make your code DRY, consider creatting a class of `d-flex` and applying it to the index.html. Currently `display: flex;` is appearing 9 times in your code base. You can read more about DRY CSS [here.](https://teamtreehouse.com/library/dry-css)

**Review by t-yanick — CHANGES_REQUESTED** [review]
Hi @selma-belhadj, 

Good job so far!

**To Highlight**

- Linters all passed.
- Use of HTML/CSS best practices
- Design matches Figma design
- Descriptive README, branch name, PR title and PR summary

There are some issues that you still need to work on to go to the next project but you are almost there!




## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

**You can also consider:**
- Beginning in future all you commit messages with capital letters and always write them in the imperative tense.
- Mentioning in your README how to open your project on the browser.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: styles.css (hunk 2 above)
Comment (t-yanick) [reply styles.css:124]: - [ ] Check line 161 of your CSS file and remove the commented code.

Comment (selma-belhadj) [reply styles.css:124]: Thank you for your review. Everything is solved

**Review by Rustamxon7 — APPROVED** [review]
## Hi @selma-belhadj   👋, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
## Congratulations! 🎉 🎉 🎉 🎉 🎉 

Cheers and Happy coding!👏👏👏

slowmo reaction

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

**Pull request merged**